core/src/actions/groups.ts

```diff
@@ -4,7 +4,7 @@ import { Group, GROUP_RULE_LIMIT, TopLevelGroupRules } from "../models/Group";
 import { ProfilePropertyRuleOpsDictionary } from "../modules/RuleOpsDictionary";
```

Comment:
is this part of this PR?

Reply:
Yeah, because I moved the logic out of the action into the op so the new status command could use it as well.